Common/DataModel/Qvectors.h

```diff
@@ -43,12 +43,12 @@ DECLARE_SOA_COLUMN(QvecFT0MReVec, qvecFT0MReVec, std::vector<float>);
 DECLARE_SOA_COLUMN(QvecFT0MImVec, qvecFT0MImVec, std::vector<float>);
 DECLARE_SOA_COLUMN(QvecFV0AReVec, qvecFV0AReVec, std::vector<float>);
 DECLARE_SOA_COLUMN(QvecFV0AImVec, qvecFV0AImVec, std::vector<float>);
-DECLARE_SOA_COLUMN(QvecBPosReVec, qvecBPosReVec, std::vector<float>);
-DECLARE_SOA_COLUMN(QvecBPosImVec, qvecBPosImVec, std::vector<float>);
-DECLARE_SOA_COLUMN(QvecBNegReVec, qvecBNegReVec, std::vector<float>);
-DECLARE_SOA_COLUMN(QvecBNegImVec, qvecBNegImVec, std::vector<float>);
-DECLARE_SOA_COLUMN(QvecBTotReVec, qvecBTotReVec, std::vector<float>);
-DECLARE_SOA_COLUMN(QvecBTotImVec, qvecBTotImVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCposReVec, qvecTPCposReVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCposImVec, qvecTPCposImVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCnegReVec, qvecTPCnegReVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCnegImVec, qvecTPCnegImVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCallReVec, qvecTPCallReVec, std::vector<float>);
+DECLARE_SOA_COLUMN(QvecTPCallImVec, qvecTPCallImVec, std::vector<float>);
```

Comment:
Please use proper camelCase for good readability. `Qvec`, `Cpos`, `Cneg`, `Call` are not words nor abbreviations.
```suggestion
DECLARE_SOA_COLUMN(QVecTpcPosReVec, qVecTpcPosReVec, std::vector<float>);
DECLARE_SOA_COLUMN(QVecTpcPosImVec, qVecTpcPosImVec, std::vector<float>);
DECLARE_SOA_COLUMN(QVecTpcNegReVec, qVecTpcNegReVec, std::vector<float>);
DECLARE_SOA_COLUMN(QVecTpcNegImVec, qVecTpcNegImVec, std::vector<float>);
DECLARE_SOA_COLUMN(QVecTpcAllReVec, qVecTpcAllReVec, std::vector<float>);
DECLARE_SOA_COLUMN(QVecTpcAllImVec, qVecTpcAllImVec, std::vector<float>);
```

Reply:
This applied to all other name changes too.